### DISCUSSION TITLE
About mern-admin

### DISCUSSION THREAD

**Discussion by marwaneTRK** [sanidhyy/mern-admin#7] [General]
The page keep loading, i cant do anything, it can be fix but I'm lost

**Comment by sanidhyy** [comment]
Yeh! You must have installed new version of dependencies. I'm sorry this was caused due to dependabot automatic updates which results in unwanted errors. Retry using an older commit c40036e and it will work fine.

Commit Link: https://github.com/Technical-Shubham-tech/mern-admin/tree/c40036eb6eae3aa0774d9cc86871786381f5a093

**Reply by sanidhyy** [reply]
Alright, I will try my best to explain:
Btw here is a GitHub beginners guide for reference: https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners

1. Clone this repository into local machine using `git clone https://github.com/Technical-Shubham-tech/mern-admin.git)`.
2. Change to older commit using command `git checkout c40036e`.
3. Now old version of this repo is loaded.
4. Just follow the steps as given in `README.md`.

NOTE: Make sure while installing dependencies from `package.json` in both **backend** and **frontend**, use this command `npm install --legacy-peer-deps`.

Hope it helps 😄
Let me know if you need any help.

**Reply by marwaneTRK** [reply]
thaaaank you budy , its working perfectly , still ony little thing , i didnt find the component that handle "shelly" "Pharmacist" in which  folder

**Reply by sanidhyy** [reply]
I haven't created a separate component for it, Its inside `components/Navbar.jsx`.

https://github.com/Technical-Shubham-tech/mern-admin/blob/dcb1bf6ea582936c91843c2b0d3370639dd47518/client/src/components/Navbar.jsx#L107

**Reply by Abhi132004** [reply]
git checkout c40036e is not working

**Reply by sanidhyy** [reply]
> git checkout c40036e is not working

Can u describe your issue and share some screenshot?
